tests/queries/0_stateless/01595_countMatches.sql

```diff
@@ -25,5 +25,10 @@ select countMatchesCaseInsensitive('foo.com BAR.COM baz.com bam.com', '([^. ]+)\
 select countMatchesCaseInsensitive('foo.com@foo.com bar.com@foo.com BAZ.com@foo.com bam.com@foo.com', '([^. ]+)\.([^. ]+)@([^. ]+)\.([^. ]+)');
 
 select 'errors';
-select countMatches(1, 'foo') from numbers(1); -- { serverError 43 }
-select countMatches('foobarfoo', toString(number)) from numbers(1); -- { serverError 44 }
+select countMatches(1, 'foo') from numbers(1); -- { serverError ILLEGAL_TYPE_OF_ARGUMENT }
+select countMatches('foobarfoo', toString(number)) from numbers(1); -- { serverError ILLEGAL_COLUMN }
+select countMatches('foo', materialize('foo')); -- { serverError ILLEGAL_COLUMN }
+
+select 'FixedString';
+select countMatches(toFixedString('foobarfoo', 9), 'foo');
+select countMatches(materialize(toFixedString('foobarfoo', 9)), 'foo');
```

Comment:
Please add tests for `String` and a constant, e.g.
``` sql
select countMatches(materialize(toString('fizzbuzz')), toString('fizz'));
select countMatches(materialize('fizzbuzz'), 'fizz');
```

Reply:
Done